tools/db_crashtest.py

```diff
@@ -365,6 +365,9 @@ def is_direct_io_supported(dbname):
     "enable_blob_files": 0,
     "use_blob_db": 0,
     "ingest_external_file_one_in": 0,
+    # TODO akanksha: Currently subcompactions is failing with user_defined_timestamp.
```

Comment:
We also need to make sure `compact_pri` is not 4 `kRoundRobin`.

Reply:
Maybe add a comment about it as well.

Reply:
Oh I didn't realize the failure occurs only if subcompaction is greater than 1 and compact_pri != 4

Reply:
The failure can happen if:
- subcompaction > 1, **or**
- compact_pri == 4, because in this case, even if subcompactions is 0, there can still be multiple subcompactions.

Reply:
I set the compact_pri = 3 kMinOverlappingRatio and subcompactions = 4 and it failed.

Reply:
ah ok. Got it.